### ISSUE TITLE
Bouygues konnectors are broken

### ISSUE THREAD

**Issue by Phyks** [cozy-labs/konnectors#498]
Hi,

Bouygues website has been updated (recently?) and the associated konnectors are broken now.

First of all, an extra field, the last name, is required to login to the user account, so the konnector should be extended to support this new field.

Then, after connection, the page that used to contain all the bills is now completely dynamic and data is fetched from a remote API behind authentication. I did not yet have found where the authentication token can be found. Good point is that once we can get access to this API, there seems to be pretty much everything there, in JSON format, so that would be really cool.

Note that weboob modules for Bouygues are also broken at the moment. There is a [related issue](https://symlink.me/issues/3542) (posting the link to the issue so that we can backport it if weboob comes with a solution first).

**Comment by eravion** [comment]
Hello,

Konnector logs looks ok, but no bills are downloaded : 
[2016-10-27 20:43:16:820] info - Bouygues Telecom | Successfully logged in.
[2016-10-27 20:43:17:42] info - Bouygues Telecom | Download bill HTML page...
[2016-10-27 20:43:17:610] info - Bouygues Telecom | Bill page downloaded.
[2016-10-27 20:43:17:616] info - Bouygues Telecom | Bill data parsed.
[2016-10-27 20:43:17:648] info - Bouygues Telecom | Import finished
[2016-10-27 20:43:17:753] info - Konnector metadata updated.

Any news about it ?

**Comment by ZeHiro** [comment]
@eravion looks like the page where the bills are displayed have evolved...

**Comment by doubleface** [comment]
Actually, this  konnector still works for me. I removed all the bills from files and rerun the bouyguestelecom konnector and the bills came back.

**Comment by eravion** [comment]
Sorry not working for me.
@doubleface : Are you downloading Mobile bills ? I tried to change the folder by adding a new one and it is not working. I also tried to remove from original folder old bill, it is nok too.

**Comment by doubleface** [comment]
Yes it is mobile billd. But I just saw that the Bouygues box konnector does not work. Some code in it did not evolve with the mobile one and the back end is the same.

But this will not solve your problem.

**Comment by Phyks** [comment]
Same, am using the "box" connector. Mobile connector may work, but won't fetch internet bills :/

**Comment by ZeHiro** [comment]
@Phyks As the client is currently being rewritten, adding new inputs does not seem 'a good idea', furthermore, this would mean displaying 2 folders to the user. Having bouygues konnector called by bouygues tel and bouygues internet would be easier I think.

**Comment by Phyks** [comment]
@ZeHiro I was just thinking in adding another [field](https://github.com/cozy-labs/konnectors/blob/master/server/konnectors/bouyguestelecom.coffee#L43) actually, but I got your point. 

Anyways, not sure this is the best issue to discuss this UI issue, but I can remember it already appeared in another issue, and might as well be closely related to Weboob integration.
